content/clients/_index.md

```diff
@@ -44,7 +44,7 @@ justify = "center"
         <a href="https://www.microsoft.com/store/apps/9P2DRTG62QF8" class="button button__accent button__card">Go to Microsoft Store</a>
     </div>
     <div class="card">
-        <img src="/images/clients/playstation_4.svg"/>
+        <img src="/images/clients/playstation-4.svg"/>
```

Comment:
do you have a personal vendetta against underscores?

Reply:
Yes, but that's not the reason I changed this. We have another image using dashes, so picking one or the other would be a good idea. My preference for dashes resulted in this change.